### ISSUE TITLE
Help with plain websockets

### ISSUE THREAD

**Issue by NathanFrund** [kit-clj/kit#89]
Hello,

I have a base Kit-CLJ project up and running. It's fantastic.

I've reached the point where I require a straightforward websocket endpoint, but I'm unsure of how to integrate that. That is, without interfering with the automatic module installation process, defining the route, etc.

Am I correct in thinking that I should start with kit-undertow?

Thanks!

**Comment by yogthos** [comment]
Yeah, using kit-undertow would be the easiest option. If you want to do this manually, you can just use the module as a reference here https://github.com/kit-clj/modules/tree/master/sente/assets

**Comment by NathanFrund** [comment]
Thank you, Dmitri.

That'll get me started!
